core/app/views/spree/order_mailer/inventory_cancellation_email.html.erb

```diff
@@ -0,0 +1,26 @@
+<table>
+  <tr>
+    <td>
+      <p class="lede">
+        <%= Spree.t('order_mailer.inventory_cancellation.dear_customer') %>
+      </p>
+      <p>
+        <%= Spree.t('order_mailer.inventory_cancellation.instructions') %>
+      </p>
+      <p>
+        <%= Spree.t('order_mailer.inventory_cancellation.order_summary_canceled') %>
+      </p>
+      <table>
+        <% @inventory_units.each do |item| %>
+          <tr>
+            <td><%= item.variant.sku %></td>
+            <td><%= raw(item.variant.product.name) %></td>
+            <td><%= raw(item.variant.options_text) -%></td>
```

Comment:
If text formatting is the reason for using `raw`, we probably should use `sanitize` here.